.ci.yaml

```diff
@@ -84,7 +84,7 @@ platform_properties:
       dependencies: >-
         [
           {"dependency": "android_sdk", "version": "version:29.0"},
-          {"dependency": "chrome_and_driver", "version": "kkOzlCHhXTi3TpU7miUzqSOCVa_2RJVb09kfze0a6z0C"},
+          {"dependency": "chrome_and_driver", "version": "latest-96"},
```

Comment:
In this case, what does latest indicate? Are there multiple releases of 96?

Reply:
I'm not sure. I'm reusing the one we already have: https://chrome-infra-packages.appspot.com/p/flutter_internal/browsers/chrome/linux-amd64/+/

Reply:
I'm finding out from @hterkelsen if I should maybe use `927388` instead.

Reply:
There's also https://chrome-infra-packages.appspot.com/p/flutter_internal/browsers/chrome/linux-amd64/+/version:96

Reply:
I think `verson: 96` is the way to go. We have been making other cipd packages following this consistent format, like android version: https://chrome-infra-packages.appspot.com/p/flutter_internal/android/sdk/linux-amd64/+/56xIdLBDFMMlbbbtLk_9czg8NB-4GjFIbxf5h3-BixQC
/cc @godofredoc

Reply:
That's correct, we are moving towards pinning explicit versions in the ci.yaml file.